mslib/mscolab/file_manager.py

```diff
@@ -250,6 +250,31 @@ def modify_user(self, user, attribute=None, value=None, action=None):
             db.session.commit()
         return True
 
+    def save_user_profile_image(self, user_id, image_data):
+        """
+        Save the user's profile image to the database.
+        """
+        user = User.query.get(user_id)
+        if user:
+            try:
+                user.profile_image = image_data
```

Comment:
here we should somehow check what we are saving

Reply:
it maybe also be better to store the file on the disk and the reference of the path in the database
see the discussion on

https://stackoverflow.com/questions/3748/storing-images-in-db-yea-or-nay

Reply:
over the server a user can add malicious content into the database.